x/delayedack/keeper/rollapp_packet.go

```diff
@@ -25,6 +25,13 @@ func (k Keeper) SetRollappPacket(ctx sdk.Context, rollappPacket types.RollappPac
 		rollappPacket.ProofHeight,
 		*rollappPacket.Packet,
 	), b)
+
+	ctx.EventManager().EmitEvent(
```

Comment:
I think it makes more sense to send this event once the packet is finalized.

this way, the indexer can correlate between the `transfer` event, and have the `channel` data from this event